Parsers/ASimWebSession/Parsers/vimWebSessionSquidProxy.yaml

```diff
@@ -111,7 +117,12 @@ ParserQuery: |
     | extend    
       DstHostname = tostring(DstFQDNparts[0]),
       DstDomain = strcat_array(array_slice(DstFQDNparts,1,-1),"."),
-      DstDomainType = "FQDN"
+      DstDomainType = "FQDN",
+      ASimMatchingIpAddr = case(array_length(ipaddr_has_any_prefix) == 0 ,"",
```

Comment:
You should really do the case as post filtering and filter on ASimMatchingIpAddr != "", to reduce the number of calls to has_any_ipv4_prefix

Reply:
Sure, that said, using the same method for running has_any_ipv4_prefix less times you suggested for Zscaler works here as well.